contracts/types/augmentations.d.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable @typescript-eslint/no-explicit-any
```

Comment:
Do we need this file? In past projects I've removed it and it seems to not change anything, so I don't quite understand what it does?

Reply:
looks like it augments the mocha context with contract types. see https://gist.github.com/PaulRBerg/0a8b4806898d81685719d7f7a26afff0

That being said, i don't think we usually use the mocha context, so i'll delete

Reply:
actually using the mocha context here does look a bit cleaner than

```js
let token;

//

token = await erc20.deploy(...)
```